app-modules/notifications/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "canyon-gbs/notifications",
+  "description": "",
+  "type": "library",
+  "version": "1.0",
+  "license": "proprietary",
+  "require": {
+    "filament/filament": "^3.0.0"
```

Comment:
Per Dan's recommendation, let's change this to `^3.0-stable` for now. I have done so for the other Modules in a branch I will submit a PR for today.